h5pxblock/h5pxblock.py

```diff
@@ -310,7 +341,9 @@ def result_handler(self, data, suffix=''):
                 save_score = True
             except BaseException as exp:
                 log.error("Error while publishing score %s", exp)
-
+            if save_score:
+                self.raw_score = data['result']['score']['raw']
```

Comment:
We should save the score only if it is higher than the current score, as is done in grading.

Reply:
I updated it. Could you test again?

Reply:
It's now solved!